website/content/en/preview/tasks/pod-density.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control Pod Density"
+linkTitle: "Control Pod Density"
+weight: 20
+---
+
+Pod density is the number of pods per node. 
+
+Kubernetes has a default limit of 110 pods per node. Karpener shares this default. 
```

Comment:
> Karpenter shares this default.

It doesn't currently, as it is currently using the defaults in the EKS optimized AMI which uses ENI limits. This was the reason to add the recent flag to change the pod density as we wanted a greater density that the ENI-based limits.

Reply:
Hm, I got this from @ellistarn quite literally https://github.com/aws/karpenter/issues/1216#issuecomment-1022615346

Perhaps he can clear this up?

Reply:
I think there was some confusion earlier @geoffcline. Mike's correct above.